tests/conftest.py

```diff
@@ -776,47 +767,38 @@ def dummy_climate_data_varying_canopy(fixture_core_components, fixture_empty_arr
     data = Data(grid)
 
     # Reference data
-    data["air_temperature_ref"] = DataArray(
-        np.full((3, 3), 30.0),
-        dims=["cell_id", "time_index"],
-    )
-    data["wind_speed_ref"] = DataArray(
-        np.full((3, 3), 1.0),
-        dims=["time_index", "cell_id"],
-    )
-    data["mean_annual_temperature"] = DataArray(
-        np.full((3), 20.0),
-        dims=["cell_id"],
-    )
-    data["relative_humidity_ref"] = DataArray(
-        np.full((3, 3), 90.0),
-        dims=["cell_id", "time_index"],
-    )
-    data["vapour_pressure_deficit_ref"] = DataArray(
-        np.full((3, 3), 0.14),
-        dims=["cell_id", "time_index"],
-    )
-    data["vapour_pressure_ref"] = DataArray(
-        np.full((3, 3), 0.14),
-        dims=["cell_id", "time_index"],
-    )
-    data["atmospheric_pressure_ref"] = DataArray(
-        np.full((3, 3), 96.0),
-        dims=["cell_id", "time_index"],
-    )
-    data["atmospheric_co2_ref"] = DataArray(
-        np.full((3, 3), 400.0),
-        dims=["cell_id", "time_index"],
-    )
-    data["precipitation"] = DataArray(
-        np.full((3, 3), 200.0),
-        dims=["time_index", "cell_id"],
-    )
+    ref_values = {
+        "air_temperature_ref": 30.0,
+        "air_temperature_mean_ref": 30.0,
+        "air_temperature_min_ref": 25.0,
+        "air_temperature_max_ref": 35.0,
+        "wind_speed_ref": 1.0,
+        "mean_annual_temperature": 20.0,
+        "relative_humidity_ref": 90.0,
+        "relative_humidity_mean_ref": 90.0,
```

Comment:
```suggestion
        "relative_humidity_ref": 90.0,
        "relative_humidity_ref_mean": 90.0,
```

Are these aliases of the same value? With the intention that we retire the first one, once we roll it out to all models?

I think I prefer ending the variable name with the `mean`, `min` and `max`? That then seems more consistent with the usage for `air_temperature_mean` within layers?